x-pack/solutions/security/packages/kbn-cloud-security-posture/graph/src/components/node/label_node/label_node.tsx

```diff
@@ -166,9 +166,7 @@ export const LabelNode = memo<NodeProps>((props: NodeProps) => {
           />
         </LabelNodeContainer>
       </EuiToolTip>
-      {ips && ips?.length > 0 && countryCodes && countryCodes.length > 0 && (
```

Comment:
I think it was ok to check that you have items instead of rendering an empty element

just missing OR statement, if you have ips OR country codes